pkg/webhooks/policyvalidation.go

```diff
@@ -47,7 +48,9 @@ func (ws *WebhookServer) policyValidation(request *v1beta1.AdmissionRequest) *v1
 			},
 		}
 	}
-
+	if response != nil && len(response.Warnings) != 0 {
```

Comment:
I don't see anywhere we return the response from `Validate` method, how did it block with warnings?

Reply:
keeping it here , since we need this for the warnings, as we are working in PR #3272

Reply:
In your PR, the response is always nil, why do we want to keep it?

BTW, should we return warnings for your case, when CRD hasn't been registered yet?

Reply:
>BTW, should we return warnings for your case, when CRD hasn't been registered yet?

IMO not required,  since we already blocking it and the error explains that kind and version is not found.

Reply:
>In your PR, the response is always nil, why do we want to keep it?

Right, we are going to use the warning checks in next PR. so thought to keep it here. we can remove as well and add in upcoming PR ?

Reply:
Removed for now.

Reply:
>Will the error be printed along with the admission response? Or just in the log?
As response,  already added the proof manifest in PR description

```
$ kubectl create -f policy.yaml 
configmap/cluster-metadata created
Error from server: error when creating "policy.yaml": admission webhook "validate-policy.kyverno.svc" denied the request: the kind defined in the any match resource is invalid, err: unable to convert GVK to GVR, [Provisioner], err: kind 'Provisioner' not found in apiVersion ''
```